doc/content/integrations/lora-cloud/geolocation.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "LoRa Cloud Geolocation"
+description: ""
+---
+
+[LoRa Cloud™ Geolocation Service](https://www.loracloud.com/portal/geolocation/home) provides simple JSON web APIs that can be easily integrated with a LoRaWAN® network or application server to enable estimating the location of LoRa®-based devices.
+
+<!--more-->
+
+In general, LoRa allows locating devices withing approximately 50m, without the need for a geolocation hardware module and no extra power consumption. Instead, the RSSI, SNR and TOA metadata that are being sent as part of your device's LoRa radio frames are used to estimate the location of your device. If your device can perform WiFi scans, scanning results can also be used to estimate your device's location. This yields a significant cost reduction compared to GPS.
+
+Also, LoRa Cloud Geolocation integration allows estimating the location of LR1110-based devices with transceiver firmware from their GNSS scans.
+
+Three query types are supported in scope of this integration and will be demonstrated below:
+
+- [LoRa® TOA/RSSI](https://www.loracloud.com/documentation/geolocation?url=v3.html) - allows computing the location estimates using TOA and/or RSSI information from a single or multiple LoRa frames (for all LoRa devices)
+- [GNSS](https://www.loracloud.com/documentation/geolocation?url=gnss.html) - location estimates are computed from the information contained in the GNSS scan result messages (NAV messages) (for LR1110 transceiver-based devices only)
+- [TOA/WiFi](https://www.loracloud.com/documentation/geolocation?url=v2.html#singleframe-wi-fi-tdoa-request) - the initial device location estimation is performed using the scanned WiFi access points, but if this estimation is not valid, another estimation is performed based on TOA metadata from device's LoRa radio frames (for all LoRa devices capable of performing WiFi scans)
+
+{{< warning >}} It is important to know that you can obtain the location estimates using LoRa Cloud Geolocation LoRa® TOA/RSSI query only if the uplink messages sent by your end device are picked up by at least 3 LoRaWAN gateways. If this is not the case, you will see no errors in {{% tts %}} Console, but you will not see the `Location solved` message containing the location estimation results either. {{</ warning >}}
+
+{{< note >}} The location of your gateway has to be publicly available for this integration.
```

Comment:
Not sure if this is within the scope of the issue, but at some point we may want to explain why this is required. 

For reference - infrastructure (gateways) is not always provided by the same entity as the a application (which contains end devices). As such, we don't reveal the location of the infrastructure unless the owner explicitly requests so. It is safe to enable if the infrastructure is owned by the application owner too.